### PR TITLE
feat: `Open in Editor` action

### PR DESCRIPTION
**Pull request by olerichter00**
This PR adds an `Open in Editor` action and preferences to configure the project's root directory and the preferred editor.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,6 +15,25 @@
...
+      "name": "editor",
+      "type": "textfield",
+      "required": true,
+      "title": "Editor",
+      "description": "The editor to use for opening repositories.",
+      "placeholder": "Enter the name of your editor",
+      "default": "com.microsoft.VSCode"
+    },
+    {
+      "name": "project_root_path",
```
Comment (jonallured) [inline package.json:29]: This is a really good idea! 👍

**Review by jonallured — APPROVED** [review]
As a terminal vim user I don't think I'll use this but I can see the appeal for vscode users!

**Comment by olerichter00** [comment]
> As a terminal vim user I don't think I'll use this but I can see the appeal for vscode users!

I wonder if you could open the project in Vim with this command. 🤔 I use it mainly to switch between already opened editor windows.

**Comment by jonallured** [comment]
I think you could if you used MacVim but probably not when using vim inside of iTerm via tmux. 🤷

**Pull request merged**